docs/guides/run-primitives-rest-api.mdx

```diff
@@ -0,0 +1,489 @@
+---
+title: Run primitives with REST API
```

Comment:
I was just speaking with Jessie about the appropriate phrasing for using primitives, she said we tend to talk about primitives in terms of "querying" rather than "running" a primitive. So I think for the title of this page we should just drop the "run" and use the title "Primitives with REST API" (it also takes up less space in the sidebar)